core/src/main/clojure/xtdb/types.clj

```diff
@@ -1212,13 +1313,19 @@
    [:timestamp-tz :micro] :timestamptz
    :tstz-range :tstz-range
    [:interval :month-day-nano] :interval
+   [:list :i32] :_int4
+   [:list :i64] :_int8
 
-   #_#_ ; FIXME not supported by pgjdbc until we sort #3683 and #3212
-   :transit :transit})
+   #_#_; FIXME not supported by pgjdbc until we sort #3683 and #3212
+       :transit :transit})
 
 (defn col-type->pg-type [fallback-pg-type col-type]
   (get col-types->pg-types
        (cond-> col-type
+         ;; ignore lists
+         (is-list-type col-type)
+         (subvec 0 2)
```

Comment:
I don't think you want to ignore arbitrary lists here.

Reply:
fair enough - think this must have been an overhang from before.